src/trade.js

```diff
@@ -22,6 +22,8 @@ class Trade {
 
   get createdAt () { return this._createdAt; }
 
+  get expiresAt () { return this._expiresAt; }
```

Comment:
Can you add a test? Also, can you set a default value for `_expiresAt` in the constructor? You can make the default `null` if you like; not all partners will have an expiration date for trades.